### DISCUSSION TITLE
WonderSwan CRC sd file

### DISCUSSION THREAD

**Discussion by PsyK0p4T** [sanni/cartreader#743] [Look what I made]
Updated the WS.ino
If the internal checksum matches at the end of dumping, it will also goes for CRC32 check / rom renaming.
It will work with both WonderSwan and Benesse Pocket Challenge v2 carts.

I only need someone to generate the "ws.txt" file for sd,  with all ".ws", ".wsc" and ".pc2" filenames + hashes.
You can use the sample file attached as a model.

[WS.zip](https://github.com/sanni/cartreader/files/11056926/WS.zip)

**Comment by sanni** [comment]
Here you go:
[wsdata.zip](https://github.com/sanni/cartreader/files/11060215/wsdata.zip)

**Reply by PsyK0p4T** [reply]
Thanks a lot, commited :)
